server/aws-lsp-codewhisperer/src/language-server/agenticChat/context/additionalContextProvider.ts

```diff
@@ -176,11 +176,7 @@ export class AdditionalContextProvider {
             return 'code'
         }
         if (prompt.filePath.endsWith(promptFileExtension)) {
-            if (
-                pathUtils.isInDirectory(path.join('.amazonq', 'rules'), prompt.relativePath) ||
-                path.basename(prompt.relativePath) === 'AmazonQ.md' ||
-                path.basename(prompt.relativePath) === 'README.md'
-            ) {
+            if (pathUtils.isInDirectory(path.join('.amazonq', 'rules'), prompt.relativePath)) {
```

Comment:
Can you justify the removal of AmazonQ.md files?

Reply:
My understanding is that these files are meant to be included just for context, only files in the .amazonq/rules folder should be prompt instruction. If that is wrong let me know